src/AudioUnitSDK/AUMIDIBase.cpp

```diff
@@ -159,7 +159,7 @@ OSStatus AUMIDIBase::DelegateSetProperty(AudioUnitPropertyID inID, AudioUnitScop
 	}
 }
 
-constexpr uint8_t MIDIStatusNibbleValue(uint8_t status) noexcept { return (status & 0xF0U) >> 4u; }
+constexpr uint8_t MIDIStatusNibbleValue(uint8_t status) noexcept { return status >> 4u; }
```

Comment:
why?

Reply:
Answer: the & 0xF0U is redundant since the right shift will take care of it

Reply:
True, though perhaps this comes down to some stylistic preferences. The way it is currently is arguably more explicit about the intention behind and expectations of the operations.